src/gui_common/dialogs/CustomDialog.cs

```diff
@@ -395,6 +442,38 @@ protected override void ApplyRectSettings()
         }
     }
 
+    protected void ConnectToWindowReorderingNodes()
+    {
+        var windowReorderingAncestorsIEnumerable = AddWindowReorderingSupportToSiblings.GetWindowReorderingAncestors(
+            this, AutomaticWindowReorderingDepth, WindowReorderingPaths);
+
+        foreach (var windowReordering in windowReorderingAncestorsIEnumerable)
+        {
+            windowReordering.ReorderingNode.ConnectWindow(
+                this, windowReordering.Sibling, AllowWindowReorderingRecursion);
+            windowReorderingNodes.Add(windowReordering.ReorderingNode);
+        }
+    }
+
+    protected void DisconnectFromWindowReorderingNodes()
```

Comment:
Shouldn't this and the above method be private as these are not meant to be called by derived classes?

Reply:
You are right.